### ISSUE TITLE
Accessibility issue on some primeng components checkbox and dropdown and date : Missing form label(A form control does not have a corresponding label)

### ISSUE THREAD

**Issue by Inaamgharbi** [primefaces/primeng#10974]
Adds aria-label to checkbox and combobox  and date
Wave : error : Missing form label
A form control does not have a corresponding label

Angular version: 12
Windows 10 Pro, VSCode

PrimeNG version: 12

Browser: all

Language: TypeScript

**Comment by yigitfindikli** [comment]
All accessibility issues will be fixed in 2022 Q1. We're gonna publish accessibility scores as well. Thanks for your report.
